### DISCUSSION TITLE
vrpn for Crazyswarm and Dtrack2 (Duplicated Marker Arrangements )

### DISCUSSION THREAD

**Discussion by TagirMuslimov** [USC-ACTLab/crazyswarm#656] [Q&A]
Hello. I am trying to use vrpn for the Crazyswarm project. I am using ART cameras and DTrack2. Is it possible to get data as raw pointcloud data? I need it to use Duplicated Marker Arrangements in Crazyswarm.

**Comment by whoenig** [comment]
Our libmotioncapture VRPN backend does not support access of (un)labelled markers, unfortunately. According to DTrack, it is possible (https://ar-tracking.com/system/files/2022-02/Supported%20SW%20Interfaces.pdf) in principle. If you get some VRPN example code, I could help you adding this feature to libmotioncapture.
